lab-steven/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "lab-steven",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "DEBUG='photogram*' mocha",
+    "start": "DEBUG='photogram*' node server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "aws-sdk": "^2.7.21",
+    "bcrypt": "^1.0.2",
+    "bluebird": "^3.4.7",
+    "body-parser": "^1.15.2",
+    "cors": "^2.8.1",
+    "crypto": "0.0.3",
```

Comment:
You don't need to have `crypto` here, it's [built in to node](https://nodejs.org/api/crypto.html).